### PR TITLE
Modernization of project and Updates for 1.3

### PR DESCRIPTION
**Pull request by jamesbascle**
Hi,

I have updated Prepare Carefully for 1.3.  It seems to work fine from what I can tell.  I didn't do super thorough testing, but I messed with all the features I could find and made sure it generated the characters into the game correctly without blowing up.

To make development easier I upgraded the project to the new SDK style project format and added Harmony and the Rimworld libraries as NuGet package dependencies.  This made redundant a number of features of the repo, including explicit includes in the .csproj file, getting rid of the libraries directory entirely, and getting rid of the AssemblyInfo.cs file.

Speaking of the AssemblyInfo.cs file, I copied the settings into the new .csproj package area, but was unsure of what exactly to update the AssemblyVersion and AssemblyFileVersion values to for this PR, so I left them as is.

### CODE REVIEW

File: Source/CustomMapInitData.cs [hunk 1]
```diff
@@ -1,234 +0,0 @@
-﻿using RimWorld;
```
Comment (jamesbascle) [inline Source/CustomMapInitData.cs:1]: This file seemed to be dead code that was using non-existent types such as StandardEquipment.

**Comment by Osyruu** [comment]
Don't work either ):

**Comment by jamesbascle** [comment]
Yeah. It broke in one of the mini updates since this PR, and I also found a weird UI bug that when you click the Needs tab it makes the other tabs go away and doesn't even show the Needs stuff.  I wasn't able to trace down the cause and the author seems MIA for now, so I am just using Character Editor and trying to keep my characters power levels in check manually.

**Comment by TheBando** [comment]
do you have a log from when it broke?

**Comment by mmoyneur** [comment]
> Hi,
> 
> I have updated Prepare Carefully for 1.3. It seems to work fine from what I can tell. I didn't do super thorough testing, but I messed with all the features I could find and made sure it generated the characters into the game correctly without blowing up.
> 
> To make development easier I upgraded the project to the new SDK style project format and added Harmony and the Rimworld libraries as NuGet package dependencies. This made redundant a number of features of the repo, including explicit includes in the .csproj file, getting rid of the libraries directory entirely, and getting rid of the AssemblyInfo.cs file.
> 
> Speaking of the AssemblyInfo.cs file, I copied the settings into the new .csproj package area, but was unsure of what exactly to update the AssemblyVersion and AssemblyFileVersion values to for this PR, so I left them as is.

hi im very new to coding and i love rimworld could you help me with how im supposed install this for 1.3

**Pull request closed without merging**